config/jobs/kubernetes/sig-cloud-provider/azure/release-1.16.yaml

```diff
@@ -424,6 +424,61 @@ periodics:
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Run Azure Disk e2e test with Azure Disk in-tree volume plugin on a CAPZ cluster.
     testgrid-num-columns-recent: '30'
+- interval: 24h
```

Comment:
@chewong did you want to start those with a smaller interval? or have you already verified it works?

Reply:
Sure, let me set it to 3h to get some consistent test signals first.